.github/workflows/js-tests.yml

```diff
@@ -1,22 +0,0 @@
----
-name: js-tests
-
-on: [push, pull_request]
-
-jobs:
-  test:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: deployment
-    steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js
-        uses: actions/setup-node@v1
-        with:
-          node-version: 12.x
-          cache: npm
-      - run: yarn install
```

Comment:
Was this file used for dashboard tests as well? https://github.com/opensearch-project/opensearch-build/search?q=yarn

Reply:
See line 11
```
    defaults:
      run:
        working-directory: deployment
```